src/view/edit.js

```diff
@@ -1,45 +1,61 @@
 (function(){
-  LifeGamification.edit = {};
-
-  const skillEditHTML = function (number) {
-    return (`
-    	<div class="skill">
-        <a class="skill__remove"><img src="../../assets/x.svg"class="skill__remove" id="remove${number}"></a>
-      `) + LifeGamification.home.skillHTML(number);
-  }
-
-  const appendEditSkill = function (skill) {
-    $('.all-skills').append(skillEditHTML(LifeGamification.skillsView.length));
-    LifeGamification.skillsView.push(skill);
-    LifeGamification.home.viewLevelAndExp(skill);
-  }
-
-  LifeGamification.edit.render = function (skills) {
-    for (let name in skills) {
-      appendEditSkill(skills[name]);
+  class EditSkill extends  LifeGamification.view.SkillView {
+    _addRemoveButton() {
+      const removeButton = this._appendNewElement(
+        'img', {
+          "class": "skill__remove",
+          "src": "../../assets/x.svg"
+        });
+      removeButton.onclick = () => {
+        this.removeFunc(this.model);
+      }
+    }
+    render() {
+      this._addTitle();
+      this._addRemoveButton();
+      this._addProgressBar();
+      this._addExpTillNextLevel();
     }
-    $('.add-skill').html(`
-      <textarea class="add-skill__name" placeholder="New skill name"></textarea>
-      <div class="add-skill__button">
-        <span class="add-skill__button-helper"></span><img src="../../assets/plus.svg" class="add-skill__button-icon">
-      </div>
-    `);
-    handleAddSkillButton();
   }
 
-  const handleAddSkillButton = function () {
-    const add_skill = function () {
-      const skillName = $('.add-skill__name').val();
-      $('.add-skill__name').val('');
-      LifeGamification.models.addSkill(skillName)
-        .then(appendEditSkill);
+  const EditView = class EditView extends LifeGamification.view.HomeView {
+    _addViewChild(skill) {
+      const editSkill = new EditSkill(skill);
+      editSkill.removeFunc = (skill) => this.model.remove(skill);
+      this.viewChildren.push(editSkill);
     }
 
-    $('.add-skill__button-icon').click(add_skill);
-    $('.add-skill__name').keyup(function (event) {
-      if (event.keyCode === 13) {
-        add_skill();
+    _createAddSkill () {
```

Comment:
Change name to `_createAdditionOfSkills`. I had trouble wondering whether this function creates or adds skills.